.travis.yml

```diff
@@ -1,11 +1,26 @@
-language: ruby
-rvm:
-  - 2.6.3
+language: minimal
+
+before_install:
+  - docker pull kidus60/travis-testing:latest 
```

Comment:
Are we keeping your fork in here for these?

Reply:
I guess for now, but I can just go ahead and creat a docker account for 'kabanero-website' push the image and update the pr